src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -76,7 +74,7 @@ const NavBar: React.FunctionComponent<PropsType> = ({ redirectionUrl }) => {
               />
             </span>
           </Link>
-          {user ? (
+          {profile ? (
```

Comment:
it should be user here I think. We want to see the avatar with an anonymous user picture when the user doesn't have a profile